src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -174,6 +174,8 @@ public static void deleteColony(final int id)
                     colonyWorlds.add(world);
                 }
             }
+            if(canDestroy)
+            {
             Log.getLogger().info("Removing buildings for " + id);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 17. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AIndentationCheck)